### PR TITLE
add standarized build script

### PR DESCRIPTION
**Pull request by svenwltr**
As mentioned in the stand-up today, I want to have some way of standardized building golang projects.

I had something like this in my mind. What do you guys think?

Projects would look like this, now: https://github.com/rebuy-de/kubernetes-deployment/pull/5

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/it-platform Please take a look!

**Comment by hendrik-rebuy** [comment]
:+1:

**Comment by alexsomesan** [comment]
👍  nice

**Comment by bjoernhaeuser** [comment]
@svenwltr why did you choose `assets` over `hack` or `bin`?

File: assets/build.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```
Comment (das-vinculum) [inline assets/build.sh]: Shouldn't this reside under `/bin/build.sh`?

**Comment by svenwltr** [comment]
@bjoernhaeuser Because this is a file which is copied to the image. It usually confuses me, when there is no clear distinction between helper scripts and files to copy.

File: assets/build.sh (hunk 1 above)
Comment (svenwltr) [inline assets/build.sh]: No, since it is copied to the docker image.

**Pull request merged**